articles/sso/v2/single-page-apps-sso.md

```diff
@@ -0,0 +1,177 @@
+---
+description: How to implement client-side SSO on single page applications.
+toc: true
+---
+
+# Client-Side SSO on Single Page Applications
+
+Single Page Applications (SPA) are user-friendly apps that load a single HTML page. This page then dynamically updates as the users interacts with the app. If your SPA is associated with other apps or sites that asks for authentication, you can implement OIDC-compliant Single Sign On to minimize the number of times the user has to provide their credentials. This ensures both the security of the process and ease of use from the perspective of the user.
+
+## What SSO Looks Like
+
+Suppose you have three applications:
+
+* App 1: app1.com (single page app)
+* App 2: app2.com (single page app)
+* App 3: app3.com (regular web app)
+
+If you've implemented SSO and the user logs in to *any* of the three applications, the user should automatically be logged in to the other applications.
+
+This article shows you how to implement OIDC-compliant Single Sign On via [Silent Authentication](/api-auth/tutorials/silent-authentication) for your Single Page Applications using [this sample](https://github.com/auth0-samples/oidc-sso-sample). This process involves
+
+## Implement SSO on Your SPA
```

Comment:
I don't really see why this header is here. It has only one pabel and then it goes into another header `### Get Started with the OIDC-Compliant Single Sign On Sample`. I would add only one header with sth like `Configure the Sample Application`